deps/Makefile

```diff
@@ -163,6 +163,21 @@ DEP_LIBS_STAGED := $(filter-out suitesparse suitesparse-wrapper osxunwind,$(DEP_
 ## Common build target prefixes
 
 default: | $(build_prefix)
+
+# Part of the FreeBSD libgcc_s kludge
```

Comment:
this fits more with the kind of things the top-level `Makefile` does, rather than the rest of the things `deps/Makefile` does - not a big deal but it sticks out a little